.mocharc.yaml

```diff
@@ -1,10 +1,8 @@
 exit: true
 extension:
   - ts
-file:
-  - mocha/setup.ts
 package: ./package.json
 reporter: spec
 require:
   - 'ts-node/register'
-spec: spec/**/*.spec.ts
```

Comment:
removing this just means that we now have test files outside the `spec/` dir and they don't need spec in the title, correct?

Reply:
Yes - having the spec defined here made it impossible for me to not run them for integration tests. So we now pass in the path/glob of test we want to run in the npm script definition (in package.json) itself.

Reply:
Ahhh the changed line - `"test": "mocha --file ./mocha/setup.ts spec/**/*.spec.ts ",`. Thanks!

Reply:
Hmm... there's no other way to achieve our goal? Changing mocha to need a --file means that running an individual file's tests with `npx mocha` got harder.

Reply:
Hmm don't you still need to pass individual file via `npx mocha`, e.g. `npx mocha spec/path/to/my/test.ts`?

I'm not sure what devx I just broke here. I don't know if other ways exists, but happy to explore.